src/app/page.tsx

```diff
@@ -2,7 +2,7 @@ import Image from "next/image";
 
 export default function Home() {
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
+    <main className="min-h-screen flex-col items-center justify-between p-24">
```

Comment:
Removing the 'flex' class might change the layout unexpectedly, especially if this class was intended for a responsive design or specific layout. Consider testing the layout on different screen sizes to ensure it maintains the desired structure.